src/pyne.cc

```diff
@@ -10339,7 +10339,7 @@ const double pyne::barns_per_cm2 = 1e24;
 const double pyne::cm2_per_barn = 1e-24;
 const double pyne::sec_per_day = 24.0 * 3600.0;
 const double pyne::MeV_per_K = 8.617343e-11;
-const double pyne::MeV_per_MJ = 1.60217657e-19;
+const double pyne::MeV_per_MJ = 6.2415096471204E+18;
```

Comment:
1. This is what is in PyNE, and 
2. Here is where the number comes from, http://www.translatorscafe.com/cafe/EN/units-converter/energy/9-65/megaelectron-volt-kelvin/

And note that MeV << MJ, so for x MeV /  1 MJ, x will be big.